model/event.go

```diff
@@ -219,7 +226,11 @@ WHERE
 	for _, a := range allAttendance {
 		i := eventIDToIndex[a.EventID]
 		events[i].Attendees = append(events[i].Attendees, a.ActivistName)
-		events[i].AttendeeEmails = append(events[i].AttendeeEmails, a.ActivistEmail)
+		if a.ActivistEmail != "" {
```

Comment:
I don't think this is a good idea. Before we had the guarantee that Attendees[j] and AttendeeEmails[j] referred to the same activist, but now this will be lost if an attendee in the middle of the list is missing their email.

I think it's better for the survey mailer logic to handle keeping track of attendees who we don't have an email for and thus couldn't survey.

Reply:
Fixed in next commit.